packages/terra-framework-docs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
   * Migrate docs from terra-layout
   * Added docs for `terra-filter-pills`
   * Added tests for `terra-filter-pills`
+  * Migrate docs from terra-slide-panel-manager
```

Comment:
This needs to be moved to the Unreleased section.

Reply:
changed ce75771ad11ed4565edefecb6ce9e945419108f1